app/components/ConfirmTrade.tsx

```diff
@@ -13,12 +13,14 @@ import { SLIPPAGE, SwapDirection, SwapState } from ".";
 import { useState } from "react";
 import { waitForTransactionReceipt, writeContract } from "wagmi/actions";
 import MarketABI from "@/abi/market.json";
-import { CONDITIONAL_TOKEN_CONTRACT_ADDRESS } from "@/model/conditionalTokens";
+import {
+  CONDITIONAL_TOKEN_CONTRACT_ADDRESS,
+  useReadCalcSellAmount,
```

Comment:
Minor, non-blocking
I'd move all these variants of WAGMI's `useReadContract` into another folder, maybe something called `hooks` or something like that, as they're not contracts nor markets; they're utility functions.

WDYT?

Reply:
They are related to contracts, that's why the name, but yes they are also only hooks. So hooks/contracts seems to do the trick. Thanks!